packages/web-components/@groupby/elements-autocomplete/src/autocomplete.ts

```diff
@@ -167,11 +170,14 @@ export default class Autocomplete extends Base {
 
   /**
    * Receives an event for populating initial data.
+   * Will not populate if the component has already received data
+   * from another source.
```

Comment:
Consider "This function will do nothing if the component had previously received data." At minimum, though, "will not populate" does not read well: I think you'd have to say "This function will not populate the component with data if ..."